cloudsync/api.py

```diff
@@ -252,6 +305,14 @@ def transcode_video(video, video_file, generate_mp4_videofile=False):
         ):
             video.update_status(VideoStatus.TRANSCODING)
 
+    except ClientError:
+        log.error("Transcode job creation failed", video_id=video.id)
+        if video.status == VideoStatus.RETRANSCODE_SCHEDULED:
+            video.status = VideoStatus.RETRANSCODE_FAILED
+        else:
+            video.update_status(VideoStatus.TRANSCODE_FAILED_INTERNAL)
+        video.save()
```

Comment:
`update_status` save the video object as well, we can safely remove `video.save()`